### PR TITLE
COMMONS-547 A Null pointer exception while processing UINotificationPopoverToolba…

### PR DESCRIPTION
**Pull request by AtiaZ**
…rPortlet in cluster prevents second node to start

fix description : 

- Add try/catch block to avoid the NPE.
- Change the replicated parameter's value to false.

### CODE REVIEW

**Comment by boubaker** [comment]
We have to test on user nullability in UINotificationPopoverToolBarPortlet.processRender and UINotificationPopoverToolBarPortlet.serveResource and if the remote user returned by WebuiRequestContext, use conversation state to try getting the current userId. If it's null, log a warn and return empty result.

**Comment by AtiaZ** [comment]
requested changes added in this PR https://github.com/exodev/platform/pull/235

File: commons-component-common/src/main/java/org/exoplatform/commons/notification/impl/service/WebNotificationServiceImpl.java [hunk 1]
```diff
@@ -104,6 +105,23 @@ public void resetNumberOnBadge(String userId) {
...
+    if (StringUtils.isNotBlank(userId)) {
+      try {
+        return storage.getNumberOnBadge(userId);
+      } catch (Exception e) {
+        if (LOG.isDebugEnabled()) {
+          LOG.error("Exception raising when getNumberOnBadge() ", e);
+        } else {
+          if (userId != null) {
+            LOG.warn("Exception raising when getNumberOnBadge() associated to the userId " + userId);
+          } else {
```
Comment (aboughzela) [inline commons-component-common/src/main/java/org/exoplatform/commons/notification/impl/service/WebNotificationServiceImpl.java]: move the check userId == null before storage.getNumberOnBadge(userId)

Comment (AtiaZ) [reply commons-component-common/src/main/java/org/exoplatform/commons/notification/impl/service/WebNotificationServiceImpl.java]: change added as discussed

**Pull request merged**